### ISSUE TITLE
Q: How to auto rename internal link complement

### ISSUE THREAD

**Issue by l2eused** [tadashi-aikawa/obsidian-various-complements-plugin#304]
I set **new link format** to `relative path to file` and in #116 it already impliment link to match obsidian settings.

somehow, I want to auto rename like obsidian internal link suggest does. (example: `../foo/bar` will create `[[../foo/bar|bar]]`)

I see that it has option `insert an alias that transformed from the displayed internal link` that my solve my problem. but I don't know how to use regex... I tried to learn how to code it before but no luck... I still confuse and can't learn it by myself. so I open this issue to ask question...

Thanks

tldr; I want to insert alt name when enter internal link complement. from `[[../path1/path2/filename]]` to `[[../path1/path2/filename|filename]]`

**Comment by tadashi-aikawa** [comment]
Hi, @l2eused :)

Please try to set `.+/(?<name>.+)$` 👍 

![image](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/9500018/990748a4-4b6a-4ead-8e7a-15f8a63840e9)
